content/_index.md

```diff
@@ -1,7 +1,50 @@
 ---
 title: "entrepreneurialopensource.org"
```

Comment:
We should ask if they want a more SEO friendly title

Reply:
+1 We need unique keywords and descriptions. For completeness, we should also set them in the config file: https://github.com/EclipseFdn/entrepreneurialopensource.org/blob/main/config.toml